sdk/metricsadvisor/Azure.AI.MetricsAdvisor/README.md

```diff
@@ -1,45 +1,466 @@
-[style-guide-msft]: https://docs.microsoft.com/style-guide/capitalization
-[style-guide-cloud]: https://aka.ms/azsdk/cloud-style-guide
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct][code_of_conduct]. For more information see the [Code of Conduct FAQ][coc_faq] or contact [opencode@microsoft.com][coc_contact] with any additional questions or comments.
 
 ![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-net%2Fsdk%2Fmetricsadvisor%2FAzure.AI.MetricsAdvisor%2FREADME.png)
+
+<!-- LINKS -->
+[metricsadv_client_src]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src
+<!--[metricsadv_docs]: https://docs.microsoft.com/en-us/azure/cognitive-services/metrics-advisor
+[metricsadv_nuget_package]: https://www.nuget.org/packages/Azure.AI.MetricsAdvisor-->
+[metricsadv_refdocs]: https://aka.ms/azsdk/net/docs/ref/metricsadvisor
+[metricsadv_rest_api]: https://westus2.dev.cognitive.microsoft.com/docs/services/MetricsAdvisor
+[metricsadv_samples]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/samples/README.md
+[metricsadv_web_portal]: https://metricsadvisor.azurewebsites.net
+
+[metrics_advisor_admin_client_class]: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorAdministrationClient.cs
+[metrics_advisor_client_class]: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorClient.cs
+
+[metricsadv-sample1]: https://github.com/kinelski/azure-sdk-for-net/tree/ma-readme/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Samples/Sample1_CreateDataFeedFromDataSource.cs
+[metricsadv-sample2]: https://github.com/kinelski/azure-sdk-for-net/tree/ma-readme/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Samples/Sample2_CheckIngestionStatusOfDataFeed.cs
+[metricsadv-sample3]: https://github.com/kinelski/azure-sdk-for-net/tree/ma-readme/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Samples/Sample3_CreateAnomalyDetectionConfiguration.cs
+[metricsadv-sample4]: https://github.com/kinelski/azure-sdk-for-net/tree/ma-readme/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Samples/Sample4_CreateHookForReceivingAnomalyAlerts.cs
+[metricsadv-sample5]: https://github.com/kinelski/azure-sdk-for-net/tree/ma-readme/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Samples/Sample5_CreateAnomalyAlertConfiguration.cs
+[metricsadv-sample6]: https://github.com/kinelski/azure-sdk-for-net/tree/ma-readme/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Samples/Sample6_QueryDetectedAnomaliesAndTriggeredAlerts.cs
```

Comment:
Did you intend to make these links to your fork?

Reply:
These files don't exist in master yet. I need to create another PR after this one to update them.